frontend/src/pages/Admin/Components/Faq/ManageFaq/ManageFaq.jsx

```diff
@@ -1,86 +1,166 @@
+import style from "./manage-faq.module.scss";
+import { SimpleToast } from "../../../../../components/util/Toast";
+
+export function ManageFaq(props) {
+  const [faqs, setFaqs] = useState([]);
+  const [expanded, setExpanded] = useState(false);
+  const [isFetching, setIsFetching] = useState(true);
+  const [loading, setLoading] = useState(false);
+  const [open, setOpenToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [severity, setSeverity] = useState("success");
+  const [reload, setReload] = useState(true);
+  const handleCloseToast = () => {
+    setTimeout(() => {
+      setOpenToast(false);
+    }, 500);
+  };
+  const handleChange = (panel) => (event, isExpanded) => {
+    setExpanded(isExpanded ? panel : false);
+  };
+  async function fetchAllFaq() {
+    try {
+      const response = await fetch(`${END_POINT}/getFaq`);
+      const data = await response.json();
+      console.log(data);
+      setFaqs(data.Faq);
+      setIsFetching(false);
+    } catch (err) {
+      console.log(err.message);
+      setIsFetching(false);
+    }
+  }
+
+  const deleteFaq = async (faqId) => {
+    setLoading(true);
+    const url = `${END_POINT}/deleteFaq`;
+    const body = {
+      userId: props.adminData._id,
+      faqId: faqId,
+    };
+    const headers = {
+      "Content-Type": "application/json",
     };
-    async function fetchAllFaq() {
-        try {
-            const response = await fetch(`${END_POINT}/getFaq`);
-            const data = await response.json();
-            console.log(data)
-            setFaqs(data.Faq)
-            setIsFetching(false)
-        }
-        catch (err) {
-            console.log(err.message)
-        }
+    try {
+      const response = await fetch(url, {
+        method: "PUT",
+        headers: headers,
+        body: JSON.stringify(body),
+      });
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      const data = await response.json();
+      setToastMessage(data.message);
+      setOpenToast(true);
+      setSeverity("success");
+      setReload(!reload);
+    } catch (error) {
+      setToastMessage("Failed to delete Faq");
+      setOpenToast(true);
+      setSeverity("error");
+    } finally {
+      setLoading(false);
     }
-    useEffect(() => {
-        setIsFetching(true)
-        fetchAllFaq()
-    }, [])
+  };
+  useEffect(() => {
+    fetchAllFaq();
+  }, [reload]);
 
-    return (
-        <div>
-            <h1 style={{ textAlign: "center" }}>Manage FAQ</h1>
-            <div className={style["faq"]}>
-                <div className={`${style["faq-block"]}`}>
-                    {isFetching ? (
-                        <Loader></Loader>
+  return (
+    <div>
+      <h1 style={{ textAlign: "center" }}>Manage FAQ</h1>
+      <div className={style["faq"]}>
+        <div className={`${style["faq-block"]}`}>
+          {isFetching ? (
+            <Loader />
```

Comment:
can we show the loader on the whole right card instead of just showing it on that delete button? The same way the loader appears during the fetch time like below one - 
<img width="1512" alt="Screenshot 2024-05-21 at 10 25 52 PM" src="https://github.com/HITK-TECH-Community/Community-Website/assets/44888949/5d9b847f-9fbe-4158-b9c0-7f85111623c1">